config.yaml

```diff
@@ -106,14 +106,26 @@ aliases:
     - "assembly:1"
     - "genome:1"
     - "pangenome:1"
-    - "pangenome_orthologfamily:1"
     - "taxon:1"
     - "tree:1"
     - "matrix:1"
-    - "attribute_mapping:1"
     - "genomeset:1"
-    - "indexing_errors:1"
     - "annotated_metagenome_assembly:1"
+  default_search_subobjects:
```

Comment:
When is this alias used? I think for searching "features" on the search UI page we want an alias for "genome_features:2" only